### ISSUE TITLE
`pg` custom option failing on newer versions of `node-postgres`

### ISSUE THREAD

**Issue by markstuart** [voxpelli/node-connect-pg-simple#33]
I was using the extended options so that I could specify the connection string, and supplying the `pg` instance that I was using elsewhere in the application, and recently ran into an issue when the `pg` version updated to `6.0.0`

``` js
  store: new pgSession({
    pg: datasource.connector.pg
    conString: pgUrl(datasource.settings)
  })
```

It seems that it stems from a change to the interface for `pg.connect` which you call here: https://github.com/voxpelli/node-connect-pg-simple/blob/master/index.js#L137

The `connect` function no longer takes a connection string as it's first argument, only an optional callback.

Possibly all that's needed is for the README to state the versions of `pg` that are supported for external use? Otherwise, I guess some detection of `pg` version and changing the usage may work as well?

**Comment by voxpelli** [comment]
Indeed looks like a breaking change that is incompatible with the way this module connects. Should probably accept a pool object in the future, rather than the pg module itself.

**Comment by voxpelli** [comment]
This was very long overdue, sorry for that, but now it's fixed and released as `4.0.0` 🎉 Many thanks to @i8-pi for helping out!

**Comment by markstuart** [comment]
Congrats on the 4.0.0 release :+1:
